requirements/dev.txt

```diff
@@ -41,66 +40,59 @@ edx-opaque-keys==2.0.0    # via edx-drf-extensions
+idna==2.8                 # via requests
+importlib-metadata==0.23  # via inflect, pluggy, tox
+inflect==3.0.1            # via jinja2-pluralize
 isort==4.3.21
 jinja2-pluralize==0.3.0   # via diff-cover
 jinja2==2.10.3            # via diff-cover, jinja2-pluralize
-lazy-object-proxy==1.4.2  # via astroid
+lazy-object-proxy==1.4.3  # via astroid
 lxml==4.4.1
 markupsafe==1.1.1         # via jinja2
 mccabe==0.6.1             # via pylint
 mock==3.0.5
-more-itertools==5.0.0     # via zipp
+more-itertools==7.2.0     # via zipp
 newrelic==5.2.1.129       # via edx-django-utils
 nose==1.3.7               # via django-nose
 packaging==19.2           # via caniusepython3, tox
-pathlib2==2.3.5           # via importlib-metadata
 pbr==5.4.3                # via stevedore
 pillow==6.2.1
 pip-tools==4.2.0
 pluggy==0.13.0            # via tox
 psutil==1.2.1             # via edx-django-utils, edx-drf-extensions
 py==1.8.0                 # via tox
 pycodestyle==2.5.0
-pycparser==2.19           # via cffi
 pycryptodomex==3.9.0      # via pyjwkest
-pydocstyle==3.0.0
+pydocstyle==4.0.1
 pygments==2.4.2           # via diff-cover
 pyjwkest==1.3.2           # via edx-drf-extensions
 pyjwt==1.7.1              # via djangorestframework-jwt
 pylint-celery==0.3        # via edx-lint
-pylint-django==0.11.1     # via edx-lint
+pylint-django==2.0.11     # via edx-lint
 pylint-plugin-utils==0.6  # via pylint-celery, pylint-django
-pylint==1.9.5             # via edx-lint, pylint-celery, pylint-django, pylint-plugin-utils
+pylint==2.4.2             # via edx-lint, pylint-celery, pylint-django, pylint-plugin-utils
 pymongo==3.9.0            # via edx-opaque-keys
-pyopenssl==19.0.0         # via urllib3
 pyparsing==2.4.2          # via packaging
 pysrt==1.1.1
 python-dateutil==2.8.0    # via edx-drf-extensions
 pytz==2019.3              # via django, fs
 requests==2.22.0          # via caniusepython3, coveralls, edx-drf-extensions, pyjwkest
 rest-condition==1.0.3     # via edx-drf-extensions
-scandir==1.10.0           # via pathlib2
 semantic-version==2.8.2   # via edx-drf-extensions
 shortuuid==0.5.0          # via edx-django-oauth2-provider
-singledispatch==3.4.0.3   # via astroid, pylint
-six==1.12.0               # via astroid, cryptography, diff-cover, edx-drf-extensions, edx-lint, edx-opaque-keys, fs, mock, more-itertools, packaging, pathlib2, pip-tools, pydocstyle, pyjwkest, pylint, pyopenssl, python-dateutil, singledispatch, stevedore, tox
+six==1.12.0               # via astroid, diff-cover, edx-drf-extensions, edx-lint, edx-opaque-keys, fs, mock, packaging, pip-tools, pyjwkest, python-dateutil, stevedore, tox
 snowballstemmer==2.0.0    # via pydocstyle
 stevedore==1.31.0         # via edx-opaque-keys
 toml==0.10.0              # via tox
 tox-battery==0.5.1
 tox==3.14.0
+typed-ast==1.4.0          # via astroid
 typing==3.7.4.1           # via fs
-urllib3[secure]==1.25.6   # via coveralls, requests
+urllib3==1.25.6           # via requests
```

Comment:
I don't think we want to drop the `[secure]` clarification, that brings in some extra dependencies.